include/internal/refcount.h

```diff
@@ -73,6 +73,21 @@ static __inline__ int CRYPTO_DOWN_REF(int *val, int *ret, void *lock)
         __atomic_thread_fence(__ATOMIC_ACQUIRE);
     return 1;
 }
+#  elif defined (__ICL)
```

Comment:
Is this define specific to the Windows version of the Intel compiler?

Reply:
Yes I think so.. What other define would i need for this?? win32?

Reply:
It could be `__WIN32`

Ref: https://sourceforge.net/p/predef/wiki/OperatingSystems/